fuel-chain-config/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "fuel-chain-config"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
Needs proper package headers like `fuel-core` or `fuel-txpool` in order to be publishable. I.e. `license`, `description`, `version` that matches the other crates etc..